stream-chat-android-compose/src/main/res/values-en/strings.xml

```diff
@@ -66,6 +66,13 @@
     <string name="stream_compose_edit_message">Edit Message</string>
     <string name="stream_compose_delete_message">Delete Message</string>
     <string name="stream_compose_mute_user">Mute User</string>
+    <string name="stream_compose_flag_message">Flag Message</string>
+
+    <!-- Message Action Confirmations -->
```

Comment:
Maybe "Message Action Prompts" as it's related to various prompts, not specifically confirmation states?